string/test/memcpy.c

```diff
@@ -28,7 +28,7 @@ static const struct fun
 # if __ARM_NEON
   F(__memcpy_aarch64_simd, 1)
 # endif
-# if __ARM_FEATURE_SVE
+# ifdef HAVE_SVE
   F(__memcpy_aarch64_sve, 1)
```

Comment:
to run the test it is not enough that the compiler supports SVE, the hw has to support it too.

we don't have runtime check for SVE so i'd leave the original ifdef check here (or add the runtime check: for each function specify a hwcap mask and if not all bits are set then skip that function).